datahub-gms-graphql-api/src/main/java/com/linkedin/datahub/graphql/api/DatahubGmsGraphQLApiApplication.java

```diff
@@ -0,0 +1,14 @@
+package com.linkedin.datahub.graphql.api;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.data.elasticsearch.ElasticsearchAutoConfiguration;
+import org.springframework.boot.autoconfigure.elasticsearch.rest.RestClientAutoConfiguration;
+
+@SuppressWarnings("checkstyle:HideUtilityClassConstructor")
+@SpringBootApplication(exclude = {ElasticsearchAutoConfiguration.class, RestClientAutoConfiguration.class})
+public class DatahubGmsGraphQLApiApplication {
```

Comment:
Nit: Are 'Daahub' & 'Api' necessary as part of the name here?

Consider `GmsGraphQLApplication` or something similar

Reply:
As always naming is the hard part, thanks for providing suggestions here...